xml/System.Data.Spatial/DbGeometry.xml

```diff
@@ -454,8 +454,8 @@
         <ReturnType>System.Nullable&lt;System.Int32&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets the number of elements in this DbGeometry value, if it represents a geometry collection. &lt;returns&gt;The number of elements in this geometry value, if it represents a collection of other geometry values; otherwise <see langword="null" />.&lt;/returns&gt;</summary>
-        <value>The number of elements in this DbGeometry value.</value>
+        <summary>Gets the number of elements in this DbGeometry value, if it represents a geometry collection.</summary>
+        <value>The number of elements in this geometry value, if it represents a collection of other geometry values; otherwise <see langword="null" />.</value>
```

Comment:
```suggestion
        <value>The number of elements in this geometry value, if it represents a collection of other geometry values; otherwise, <see langword="null" />.</value>
```